can/parser_pyx.pyx

```diff
@@ -45,10 +46,11 @@ cdef class CANParser:
     self.ts = {}
 
     self.can_invalid_cnt = CAN_INVALID_CNT
+    self.toyota_safety_param = -1
 
     cdef int i
-    cdef int num_msgs = self.dbc[0].num_msgs
-    for i in range(num_msgs):
+    cdef int x
```

Comment:
Not sure if this is needed, it works without defining them beforehand

Reply:
probably done to keep the code in C. you can check by generating a Cython report

Reply:
I checked both with and without the cdef, the generated code for the for loop looks the same.